Program.cs

```diff
@@ -4,15 +4,18 @@
 // Start Process
 var processStartInfo = new ProcessStartInfo
 {
-	FileName = @"BF1942.exe"
+    FileName = @"BF1942.exe"
```

Comment:
I know this may seem pretty minor but try to avoid whitespace changes (in this case, changing from tabs to spaces). There are a few cases where this has happened in this file that would be good to undo.

Reply:
Even though it is minor, thank you for bringing this change to my attention.  I think have probably developed a bad habit of using the CTRL+K, D functionality in VS 2022 with the default settings.  It sounds like this is a habit that I should break. :)  In my previous job when I was coding in the Borland Delphi 5 IDE, there wasn't any functionality similar to the CTRL+K, D command.  I will just have to train myself not to use the CTRL+K, D functionality in VS 2022 or maybe there is another setting/option that I can use.

Reply:
To be fair, `CTRL+K, D` is generally a pretty good habit... if I had an `.editorconfig` file that defined the result I wanted (VS2022 will follow whatever is defined there).

Some of my projects have it, some don't - probably should fix that up.